os_brick/initiator/connectors/lightos.py

```diff
@@ -187,66 +188,87 @@ def get_volume_paths(self, connection_properties):
         path = connection_properties['device_path']
         return [path]
 
-    @utils.trace
-    def _get_device_by_uuid(self, uuid):
-        lnk_path = "%s%s" % ("/dev/disk/by-id/nvme-uuid.", uuid)
+    def _check_device_exists_using_dev_lnk(uuid):
+        lnk_path = f"/dev/disk/by-id/nvme-uuid.{uuid}"
+        if os.path.exists(lnk_path):
+            devname = os.path.realpath(lnk_path)
```

Comment:
@yuval-lb 
if os.path.exists(lnk_path) 
does not exist we still continue to line 195 and will fail as devname is not initialised
shouldnt we return non here?

Reply:
yes you right

Reply:
pushed line 195 into the check exists "if"